package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonImporter.cs

```diff
@@ -134,15 +138,20 @@ public static class SkeletonImporter {
     public static void BuildSkinnedMesh(string meshPath,
                                         string skelPath,
                                         SkeletonSample skeleton,
-                                        SkelBindingSample meshBinding,
+                                        UsdSkelSkinningQuery skinningQuery,
                                         GameObject go,
                                         PrimMap primMap,
                                         SceneImportOptions options) {
-      string[] joints = meshBinding.joints;
+      VtTokenArray jointsAttr = new VtTokenArray();
+      skinningQuery.GetJointOrder(jointsAttr);
+      string[] joints = IntrinsicTypeConverter.FromVtArray(jointsAttr);
+
+      UsdGeomPrimvar jointWeights = skinningQuery.GetJointWeightsPrimvar();
+      UsdGeomPrimvar jointIndices = skinningQuery.GetJointIndicesPrimvar();
```

Comment:
Both of these objects may be invalid

Reply:
Does UsdGeomPrimvar have an `IsValid()` function? If so, this would be better than `IsDefined`, though IsDefined /may/ still do what you want (not 100% sure)

Reply:
There is no `IsValid` function, I tested `IsDefined` in various cases, it returns `false` when the attribute doesn't exist, return `true` otherwise. I also tried to set the attribute but with no values, in this case `IsDefine` still return `true`, and all `GetAttr`, `GetInterpolation`, etc. will return default values.